mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -2801,6 +2820,9 @@ func resourceContainerClusterRead(d *schema.ResourceData, meta interface{}) erro
 	if err := d.Set("gateway_api_config", flattenGatewayApiConfig(cluster.NetworkConfig.GatewayApiConfig)); err != nil {
 		return err
 	}
+	if err := d.Set("cluster_network_performance_config", flattenClusterNetworkPerformanceConfig(cluster.NetworkConfig.ClusterNetworkPerformanceConfig)); err != nil {
```

Comment:
I believe this line is linked to this error:

```
google-beta/services/container/resource_container_cluster.go:2762:117: cluster.NetworkConfig.ClusterNetworkPerformanceConfig undefined (type *"google.golang.org/api/container/v1beta1".NetworkConfig has no field or method ClusterNetworkPerformanceConfig)
```

[NetworkConfig](https://pkg.go.dev/google.golang.org/api/container/v1#NetworkConfig) doesn't have a ClusterNetworkPerformanceConfig field, but it does have a NetworkPerformanceConfig field that takes a ClusterNetworkPerformanceConfig pointer as a value. Is that what should go here?